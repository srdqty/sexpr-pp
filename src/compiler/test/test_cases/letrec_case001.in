(letrec (
  (x (lambda () (y)))
  (y (lambda () (x)))
  )
  (x)
  (y)
  )
(letrec () 5)
