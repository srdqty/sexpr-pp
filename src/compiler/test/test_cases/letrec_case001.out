(Letrec 2 ((FunFixArity (Arity = 0) (App (DeepRef 1 1) (ArgCount 0) (Args ))) (FunFixArity (Arity = 0) (App (DeepRef 1 0) (ArgCount 0) (Args )))) (App (ShallowRef 0) (ArgCount 0) (Args )) (App (ShallowRef 1) (ArgCount 0) (Args )))
(Letrec 0 () (Constant 5))
